models/skilledImpactRequestsModel.js

```diff
@@ -0,0 +1,73 @@
+const db = require('../database/dbConfig');
+
+const ProjectGoal = require('./projectGoalsModel.js');
+
+
+async function findSkilledImpactRequests(campaign_id) {
+    return db('project_goals')
+        .where({campaign_id})
+        .fullOuterJoin('skilled_impact_requests', 'skilled_impact_requests.id','project_goals.skilled_impact_request_id')
+        .select(
+           '*'
+        )
+        .then((returnedQuery)=>{
+            const skilledRequests = new Map();
+            for(let i = 0; i<returnedQuery.length; i++){
+                if(!skilledRequests.has(returnedQuery[i].id)){
+                    let skillAndProject = {
+                        skill: returnedQuery[i].skill,
+                        point_of_contact: returnedQuery[i].point_of_contact,
+                        welcome_message: returnedQuery[i].welcome_message,
+                        our_contribution: returnedQuery[i].our_contribution,
+                        project_goals:[]
+                    };
+                    skilledRequests.set(returnedQuery[i].id, skillAndProject);
+                }
+                let projectGoal = {
```

Comment:
`const` (although in this case you could just inline it as well)

Reply:
for sure